packages/mountable/src/options.ts

```diff
@@ -114,7 +114,7 @@ export function parseMountOptions(options: MountOptions): {
       files,
       archives,
       requirements: options.requirements || [],
-      allowedOriginsResp: options.allowedOriginsResp,
+      hostConfigResponse: options.hostConfigResponse,
```

Comment:
nit: I think it is better to call this just `hostConfig` here when treating it as an option that is exposed to the user. We will also soon add some official documentation about host config which explains what options exist.

Reply:
Agreed! Thanks :D

Reply:
Did it.

tbh, I haven't understood well about how the HostConfig API could be used by users and why it is separated from the existing config mechanism (`config.toml` or the command line arguments), so also don't know what's the best signature of this API on stlite (should it be integrated into the `streamlitConfig` field in the case of stlite, for example?).
Actually, currently this field is only used by the VSCode extension package internally so we can change the signature in the future anyway.

Reply:
so high level: host config is targeting host platforms while the app config (`config.toml`) is targeting app developers. But some aspects overlap. In a normal Streamlit setup, these mechanisms work quite differently (because we need to ensure tamper-proofness for host config). For stlite, the separation doesn't matter a lot. It probably could be merged into one. But I think it is better to keep this as two options since it later can be better connected to the dedicated pages on our documentation. Once we have something out on our docs, it might be a bit more understandable.